buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DiktatConfiguration.kt

```diff
@@ -17,14 +17,26 @@ fun Project.configureDiktat() {
     apply<DiktatGradlePlugin>()
     configure<DiktatExtension> {
         diktatConfigFile = rootProject.file("diktat-analysis.yml")
-        inputs = files("src/**/*.kt")
+        inputs = files("src/**/*.kt"
+//            , "**/*.kts"
```

Comment:
Can we apply this pattern or may be we should delete this line, since it's not in use?

Reply:
There are some false-positives, I've opened https://github.com/cqfn/diKTat/issues/834 for them. I'd say keep these as a todo for now, until it's fixed in diktat